tests/mpi/thresholds/fv_dynamics.yaml

```diff
@@ -0,0 +1,226 @@
+savepoints:
```

Comment:
Do we still need .gitkeep since we have a threshold yaml file here?

Reply:
No, deleted.